google/cloud/aiplatform/training_utils.py

```diff
@@ -15,6 +15,57 @@
 # limitations under the License.
 #
 
+import os
 
-class TrainingUtils:
-    pass
+
+class EnvironmentVariables:
+    """Passes on OS' environment variables"""
+
+    @property
+    def training_data_uri(self) -> str:
+        """
+        Returns: Cloud Storage URI of a directory intended for training data
+        """
+        return os.environ["AIP_TRAINING_DATA_URI"]
+
+    @property
+    def validation_data_uri(self) -> str:
+        """
+        Returns: Cloud Storage URI of a directory intended for validation data
+        """
+        return os.environ["AIP_VALIDATION_DATA_URI"]
+
+    @property
+    def test_data_uri(self) -> str:
+        """
+        Returns: Cloud Storage URI of a directory intended for test data
+        """
+        return os.environ["AIP_TEST_DATA_URI"]
+
+    @property
+    def model_dir(self) -> str:
+        """
+        Returns: Cloud Storage URI of a directory intended for saving model artefacts
+        """
+        return os.environ["AIP_MODEL_DIR"]
+
+    @property
+    def checkpoint_dir(self) -> str:
+        """
+        Returns: Cloud Storage URI of a directory intended for saving checkpoints
+        """
+        return os.environ["AIP_CHECKPOINT_DIR"]
+
+    @property
+    def tensorboard_log_dir(self) -> str:
+        """
+        Returns: Cloud Storage URI of a directory intended for saving TensorBoard logs
+        """
+        return os.environ["AIP_TENSORBOARD_LOG_DIR"]
+
+    @property
+    def cluster_spec(self) -> str:
```

Comment:
Additionally, add one more similar property for `TF_CONFIG`: https://cloud.google.com/ai-platform-unified/docs/training/distributed-training#tf-config